cms/templates/darklang/preview_lang.html

```diff
@@ -0,0 +1,43 @@
+## Override the default styles_version to the Pattern Library version (version 2)
+<%! main_css = "style-main-v1" %>
+<%page expression_filter="h"/>
+<%! from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import HTML, Text
+%>
+<%inherit file="../base.html" />
+
+<%block name="title">Preview Language Setting</%block>
```

Comment:
Translate all the strings in the file. For instance, here and placeholder text below.